src/components/questions/answersList.js

```diff
@@ -8,20 +8,47 @@ class AnswerList extends React.Component{
       answers: null,
       display: 2
     };
+    this.determineDisplayed = this.determineDisplayed.bind(this);
+    this.seeMoreAnswersClick = this.seeMoreAnswersClick.bind(this);
   }
 
-  makeAnswerArray(answerObj){
+  //DETERMINES NUMBER OF ANSWERS RENDERED
+  determineDisplayed(propsAnswers){
+    let keys = Object.keys(propsAnswers);
+    keys = keys.slice(0, this.state.display);
+    return keys;
+  }
 
+  //SETS NUMBER OF ANSWERS DISPLAYED TO ALL ON CLICK
+  seeMoreAnswersClick(event){
+    event.preventDefault();
+    this.setState({display: this.props.answers.length});
```

Comment:
I can't remember, but does your BRD section also call for 2 answers to be displayed at a time? If so, this would have to be revised

Reply:
i thought the same thing originally but i had to check:

VIA BRD 1.3.2.2 - 
"By default only two answers will show. The rest should be hidden. If more than two answers exist for the question, a link to “See more answers” should display below the list. Clicking on this link should expand the area below the question and display the remainder of the list."